proposals/lip-0052.md

```diff
@@ -679,6 +679,168 @@ else: # chainID == sendingChainID
         createUserEntry(senderAddress, nftID)
 ```
 
+
+### Genesis Block Processing
```

Comment:
There is a previous specs section "NFTs in Genesis Blocks" contradicting points of this section.

Reply:
indeed, forgot to remove it, gone.